crates/server/src/store/blocklist/mod.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2015-2022 Benjamin Fry <benjaminfry@me.com>
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// https://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// https://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+#![cfg(feature = "blocklist")]
+
+//! Blocklist resolver related types
+mod authority;
+
+pub use self::authority::BlocklistAuthority;
+use serde::Deserialize;
+
+/// Consult action enum.  Controls how consult lookups are handled.
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq)]
+pub enum BlocklistConsultAction {
```

Comment:
Nit: suggest deriving `Default` for this directly, and using that in the downstream implementation.